src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +161,44 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket with force flag, when bucket path is not empty', async () => {
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path) 
+            await create_json_file(bucket_defaults.path, 'test.json', {test: 'data'});
+            const delete_bucket_options = { config_root, name: bucket_defaults.name, force: true};
+            const resp = await exec_manage_cli(TYPES.BUCKET, ACTIONS.DELETE, delete_bucket_options);
+            expect(JSON.parse(resp.trim()).response.code).toBe(ManageCLIResponse.BucketDeleted.code);
+            const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_defaults.name + '.json');
+            await fs_utils.file_must_not_exist(config_path);
+        });
+
+        it('should fail - cli delete bucket when bucket path is not empty', async () => {
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path), 
+            await create_json_file(bucket_defaults.path, 'test1.json', {test: 'data'});
+            const delete_bucket_options = { config_root, name: bucket_defaults.name};
+            const resp = await exec_manage_cli(TYPES.BUCKET, ACTIONS.DELETE, delete_bucket_options);
+            expect(JSON.parse(resp.stdout).error.code).toBe(ManageCLIError.BucketDeleteForbiddenHasObjects.code);
+            const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_defaults.name + '.json');
+            await fs_utils.file_must_exist(config_path);
+        });
+
+        it('cli delete bucket force flag with valid boolean string(\'true\')', async () => {
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path) 
+            await create_json_file(bucket_defaults.path, 'test.json', {test: 'data'});
+            // force wth valid string value 'true'
+            const delete_bucket_options = { config_root, name: bucket_defaults.name, force: 'true'};
+            const resp = await exec_manage_cli(TYPES.BUCKET, ACTIONS.DELETE, delete_bucket_options);
+            expect(JSON.parse(resp.trim()).response.code).toBe(ManageCLIResponse.BucketDeleted.code);
+            const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_defaults.name + '.json');
+            await fs_utils.file_must_not_exist(config_path);
+        });
+
+        it('should fail - cli delete bucket force flag with invalid boolean string(\'nottrue\')', async () => {
+            const bucket_six = 'bucket6';
+            const delete_bucket_options = { config_root, name: bucket_six, force: 'nottrue'};
```

Comment:
Why not use the default bucket name?
```suggestion
            const delete_bucket_options = { config_root, bucket_defaults.name, force: 'nottrue'};
```